components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/internal/SAMLApplicationMgtListener.java

```diff
@@ -266,81 +145,22 @@ public void doExportServiceProvider(ServiceProvider serviceProvider, Boolean exp
                         StringWriter sw = new StringWriter();
                         jaxbMarshaller.marshal(samlSP, sw);
                         authConfig.setInboundConfiguration(sw.toString());
+                        //Have to clear the the create properties to make the import xml equal to the exported xml
+                        //can't just clear properties because properties may have added elsewhere.
+                        //TODO : find a better property clear solution
+                        authConfig.setProperties(new Property[0]);
```

Comment:
```suggestion
                        // TODO : find a better property clear solution
                        // Have to clear the create properties to make the import xml equal to the exported XML,
                        // not possible just clear properties because properties may have added elsewhere.
                        authConfig.setProperties(new Property[0]);
```